api/src/main/java/org/openmrs/User.java

```diff
@@ -35,14 +36,31 @@
  * key-value pairs for either quick info or display specific info that needs to be persisted (like
  * locale preferences, search options, etc)
  */
-public class User extends BaseChangeableOpenmrsMetadata implements java.io.Serializable, Attributable<User> {
-	
-	public static final long serialVersionUID = 2L;
+<<<<<<< HEAD
+@Entity
+@Table(name = "users")
+@Cacheable
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+/*
+*User inherits name and description properties from BaseOpenmrsMetadata but they don't exist in the users table since 
+*they are not used, because these field exist in BaseOpenmrsData and they are marked as persistent, hibernate generates 
+*a query containing these columns which of course fails to execute because of missing columns, so we introduce this 
+*dummy table containing these columns to 'please' hibernate until we fix the Class hierarchy see TRUNK-5665
+*/
+@SecondaryTable(name = "users_unused_fields")
+@AttributeOverrides({
+        @AttributeOverride(name = "name", column = @Column(table = "users_unused_fields", name = "name", insertable = false, updatable = false)),
+        @AttributeOverride(name = "description", column = @Column(table = "users_unused_fields", name = "description", insertable = false, updatable = false)) })
+public class User extends BaseOpenmrsObject  implements java.io.Serializable, Attributable<User>, Auditable, Retireable {
+=======
+public class User extends BaseOpenmrsObject implements java.io.Serializable, Attributable<User>, Auditable, Retireable {
+>>>>>>> 7f1edc92d80496f5d1c86e9e0b968292c5ebf238
```

Comment:
@jwnasambu So here, we'll want to select this version:

```
public class User extends BaseOpenmrsObject implements java.io.Serializable, Attributable<User>, Auditable, Retireable {
```

This code above is a Git conflict message. If shows two different versions of this section that git cannot resolve. These sections are divided by the `=======`. So we can treat the top as option 1 and the part underneath as option 2. `<<<<<<<` is used to indicate where the conflict begins and `>>>>>>>` indicates where it ends, so we don't want to modify anything outside of that.

In this, as I said, we want to go with option 2, so we'll get rid of the text that makes up option 1 and the conflict markers:

```suggestion
public class User extends BaseOpenmrsObject implements java.io.Serializable, Attributable<User>, Auditable, Retireable {
```